spaceship/templates/crew.html

```diff
@@ -187,7 +187,7 @@ <h5 class="modal-title">Invite Crew</h5>
                 {% include 'compose_invite_email.html' %}
             </div>
             <div class="modal-footer">
-                <button id="email-send" class="btn btn-primary" data-endpoint="{% block endpoint %}{% endblock %}" data-dismiss="modal">Send</button>
+                <button id="email-send" class="btn btn-primary" data-endpoint="/invite/{{ team.id }}" data-dismiss="modal">Send</button>
```

Comment:
@jeredw we did this as a hack because igor wanted to put the button in the modal. probably a better way is to have the `compose_invite_email.html` template be the thing that renders the modal, so the button can live along with the template it extends